pkg/apis/authentication/types.go

```diff
@@ -68,6 +74,13 @@ type TokenReviewStatus struct {
 	Authenticated bool
 	// User is the UserInfo associated with the provided token.
 	User UserInfo
+	// Audience is a single audience identifier chosen by the authenticator that
+	// is compatible with both the TokenReview and token. This identifier can be
+	// any identifier in the intersection of the TokenReviewSpec audiences and
+	// the token's audiences. A client of the TokenReview API should ensure that
```

Comment:
`A client of the TokenReview API that sets the spec.audiences field should ensure ...` might be clearer than "audience-bound TokenReviews"

describe the meaning of an empty `status.audience` field on an `authenticated: true` response (indicates the token is valid for the audience of the Kubernetes api server)